benchmarks/low_level.cpp

```diff
@@ -0,0 +1,455 @@
+// Copyright (c) 2021 MIT Digital Currency Initiative,
+//                    Federal Reserve Bank of Boston
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+/*
+    File contains benchmarks for
+    -   transactions size [1, 4] -> [1, 2]
```

Comment:
This list is pretty likely to fall out-of-date, and the resulting benchmarks executable is able to list all benchmarks grouped by their fixture. I'd suggest removing it.